src/contact.html

```diff
@@ -0,0 +1,109 @@
+  <title> Contact </title>
+  <link rel="icon" href="../photo/photo4.png" type="image/png">
+  <link rel="stylesheet" href="./style-sheet.css">
+  <script type="module" src="./contact.js"></script>
+</head>
+<body>
+  <header class="header">
+    <a href="/" class="logo"> Făgurel</a>
+    <input class="menu-btn" type="checkbox" id="menu-btn"/>
+    <label class="menu-icon" for="menu-btn"><span class="nav-icon"></span></label>
+      <ul class="menu">
+        <li><a href="index.html" class="menu-item">acasa</a></li>
+        <li><a href="magazin.html" class="menu-item">magazin</a></li>
+        <li><a href="contact.html" class="menu-item">contact</a></li>
+      </ul>
+  </header>
+  <section class="contact-section">
+    <div class="contact-bg">
+      <h2>Stupina Bălteni </h2>
+      <p class="text">Ne puteți contacta pentru comenzi sau dacă aveți întrebări despre produsele noastre apicole. Răspundem cu drag prin E-mail sau la Telefon</p>
+    </div>
+    <div class="wrapper">
+      <div id="oferta" class="hide">
+        Luna aceasta transportul este gratuit pentru cumparaturi de minim 100 RON
+      </div>
+      <div id="button">
+        Descopera oferta lunii!
+      </div>
+    </div>
+    <div class="contact-body">
+      <div class="contact-info">
+        <div>
+          <span><i class="fas fa-mobile-alt"></i></span>
+          <span>Telefon</span>
+          <span class="text">0721 003 004</span>
+        </div>
+        <div>
+          <span><i class="fas fa-envelope-open"></i></span>
+          <span>E-mail</span>
+          <span class="text">stupina@fagurel.ro</span>
+        </div>
+        <div>
+          <span><i class="fas fa-map-marker-alt"></i></span>
+          <span>Adresă </span>
+          <span class="text">Strada Pădurii, nr 8, Bălteni, Ilfov</span>
+        </div>
+        <div>
+          <span><i class="fas fa-clock"></i></span>
+          <span>Program </span>
+          <span class="text">Luni- Sâmbătă (9:00  - 18:00)</span>
+        </div>
+      </div>
+      <form id="form">
+        <label>
+          User:
+          <input type="text" placeholder="user name" />
+        </label>
+        <label>
+          Parola:
+          <input type="password" placeholder="password" />
+        </label>
+        <button> Log in </button>
+      </form>
+      <div class="contact-form">
+        <form>
+          <div>
+          <input type="text" class="form-control" placeholder="Nume">
+          <input type="text" class="form-control" placeholder="Prenume">
+          </div>
+          <div>
+          <input type="email" class="form-control" placeholder="E-mail">
+          <input type="text" class="form-control" placeholder="Telefon">
+          </div>
+          <textarea rows="5" placeholder="Lasă-ne un mesaj" class="form-control"></textarea>
+          <input type="submit" class="send-btn" value="Trimite">
+        </form>
+        <div>
+          <img src="../photo//photo13.jpg" alt="stupina bio">
```

Comment:
Vezi că ai pus `//`.
```suggestion
          <img src="../photo/photo13.jpg" alt="stupina bio">
```

Și poți pune pozele în `src/photos`.